admin/admin-domain-service/ballerina/init.bal

```diff
@@ -20,14 +20,21 @@ import ballerina/log;
 import ballerinax/postgresql;
 import ballerina/sql;
 import ballerina/http;
+import wso2/apk_common_lib as commons;
 
 configurable DatasourceConfiguration datasourceConfiguration = ?;
 final postgresql:Client|sql:Error dbClient;
 configurable ThrottlingConfiguration throttleConfig = ?;
 
 configurable int ADMIN_PORT = 9443;
 
-listener http:Listener ep0 = new (ADMIN_PORT);
+commons:IDPConfiguration idpConfiguration = {
+        publicKey:{path: "/home/wso2apk/admin/security/mg.pem"}
```

Comment:
better to get this from config.toml

Reply:
This will fix once external IDP is configurable.